lib/openshift/priority_class.rb

```diff
@@ -0,0 +1,8 @@
+require 'openshift/cluster_resource'
```

Comment:
@kasturinarra FYI, you don't really to add this for deletion if you are calling `admin.cli_exec...`